core/modules/layout/layout.module

```diff
@@ -1812,6 +1812,19 @@ function layout_context_required_by_path($path) {
   return $required_contexts[$path];
 }
 
+/**
+ * Return the current user context, which is always available.
+ */
```

Comment:
```suggestion
 * Returns the current user context, which is always available.
 *
 * @return EntityLayoutContext
 *   The layout context for the current user.
 *
 * @since 1.17.5 Function added.
 */
```